src/event/ngx_event_openssl.c

```diff
@@ -1222,6 +1246,82 @@ ngx_ssl_set_session(ngx_connection_t *c, ngx_ssl_session_t *session)
     return NGX_OK;
 }
 
+
+#if (NGX_SSL && NGX_SSL_ASYNC)
+ngx_int_t
+ngx_ssl_async_process_fds(ngx_connection_t *c)
```

Comment:
Why do not use NGX_OK and NGX_ERROR, but with 0 and 1

Reply:
Done